### DISCUSSION TITLE
Welcome to WT32-ETH01-LAN-8720-RJ45- Discussions!

### DISCUSSION THREAD

**Discussion by ldijkman** [ldijkman/WT32-ETH01-LAN-8720-RJ45-#1] [General]
WT32-ETH01 LAN 8720 wired webserver

**Comment by Witchdoc59** [comment]
None of the examples will compile.  The latest error message I'm getting is Compilation error: 'SYSTEM_EVENT_ETH_START' was not declared in this scope; did you mean 'ARDUINO_EVENT_ETH_START'?.   I've tried 2 different WT32-ETH01 boards.  I've tried several of the examples.  In one case I got an error that void WiFiEvent(WiFiEvent_t event) should not be void.  Changing all the SYSTEM_EVENT_ETH methods to ARDUINO_EVENT_ETH results in this error.  Compilation error: invalid conversion from 'int' to 'eth_phy_type_t' [-fpermissive].  I have updated all the boards and libraries in my Arduino IDE.

**Reply by jarling** [reply]
You have to use an older version of the ESP32 plugin in Arduino IDE: The last working one is 2.0.17, later version 3.0.0 won't work.

**Comment by Witchdoc59** [comment]
Where do you plug the BME 280 into?  The RTC is already on the I2C ports.  Do we use the same ports?
